src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1638,23 +1638,14 @@ internal static CollectionExpressionTypeKind GetCollectionExpressionTypeKind(CSh
                     return CollectionExpressionTypeKind.Array;
                 }
             }
```

Comment:
>} [](http://example.com/codeflow?start=12&length=1)

Shouldn't we perform more checks for multi-dimensional arrays?

Reply:
Does the spec explicitly say that MD arrays are not valid targets?

Reply:
MD arrays are not valid targets.

Reply:
Added clarification in https://github.com/dotnet/csharplang/pull/7734.